preview/Win7Msix/Win7MSIXInstaller/Protocol.cpp

```diff
@@ -52,13 +105,91 @@ HRESULT Protocol::ExecuteForAddRequest()
     return S_OK;
 }
 
+HRESULT Protocol::ExecuteForAddRequest()
+{
+    for (std::vector<ProtocolData>::iterator protocol = m_protocols.begin(); protocol != m_protocols.end(); ++protocol)

```

Comment:
auto?

Reply:
fixed here and for FTA as well

---
In reply to: [265693744](https://github.com/Microsoft/msix-packaging/pull/67#discussion_r265693744) [](ancestors = 265693744)